tests/test_erp_service.py

```diff
@@ -88,13 +88,14 @@ def __init__(self, erp):
             self.model = 'poweremail.templates'
 
         def list(self, field):
+            supported_languages = ['ca_ES', 'es_ES']
             translations = self.erp.IrTranslation.read([
                 ('name', '=', self.model+','+field),
                 ('res_id', '=', existing_template.erp_id),
             ],['lang', 'value'])
             return {
                 x['lang']: x['value']
-                for x in translations or []
+                for x in translations or [] if x['value'] and x['lang'] in supported_languages
```

Comment:
This helper is used to assert in test, doing this change is like faking the assert to be always true. You should either pick as fixture a new template that has no `en_US` translation, or adding `en_US` as supported traslation as previous version of the PR did but then change the unsupported language in `test__save_template__bodyTranslation_cloneExistingUnsupported` to be other than `en_US`